test/functional/options/defaults_spec.lua

```diff
@@ -152,7 +152,7 @@ describe('startup defaults', function()
       command('set ambiwidth=double')
       screen:expect([[
         1                        |1                       |
-        ^+--  2 lines: 2----------|+--  2 lines: 2---------|
+        ^+--  2 lines: 2----------|+--  2 lines: 2·········|
```

Comment:
With `ambiwidth=double` (line 152) this should not happen, it should be the single-byte default `-`.

Reply:
`ambiwidth` is global, it should apply to every window.